custom_commands/captureElementScreenshot.js

```diff
@@ -0,0 +1,43 @@
+const EventEmitter = require('events').EventEmitter;
+const util = require('util');
+const Jimp = require('jimp');
+const Buffer = require('buffer').Buffer;
+const promisifyCommand = require('../lib/promisify-command');
+
+/**
+ * Takes a screenshot of the visible region encompassed by the bounding rectangle
+ * of an element.
+*
+ * @link
+ * @param {string} id ID of the element to route the command to.
+ * @param {function} callback Callback function which is called with the captured screenshot as an argument.
+ * @returns {Object} The captured screenshot. This object is a Jimp (library) image instance.
+ */
+function CaptureElementScreenshot() {
```

Comment:
**question (if-minor):** I don't see usage of this function. Do we need it?

Reply:
This is the built-in visual regression testing. it is currently used, but I think we should talk about dumping it in the future. For now, it should be left.